src/mui-setup.js

```diff
@@ -2,8 +2,26 @@ import { createGenerateClassName, createMuiTheme } from '@material-ui/core/style
 
 export default function muiSetup(sproutBase) {
   if (sproutBase.overrides) {
-    sproutBase.overrides.MuiTableContainer.root.height = 'calc(100% - 52px)';
     sproutBase.overrides.MuiListItem.root['&$selected'] = {};
+
+    sproutBase.overrides.MuiTableBody = {
+      root: {
+        display: 'block',
+        overflow: 'auto',
+      },
+    };
```

Comment:
Is this the same?
```suggestion
    sproutBase.overrides.MuiTableBody.root = {
      display: 'block',
      overflow: 'auto',
    };
```

Reply:
no, not really.  'TypeError: Cannot set property 'root' of undefined' as MuiTableBody does not have object root